guide/topics/webdev/reference.md

```diff
@@ -0,0 +1,134 @@
+---
+title: Reference
+---
+You can find a reference to what each piece of HTML and CSS can do.
+
+# HTML
+
+## HTML Doctype Declaration
+
+#### Reason - 
+To Declare HTML Documents.
+
+#### Usage - 
+`<!DOCTYPE html>`
```

Comment:
Wouldn't it better to use , a code block here ? it would even give a syntax highlighting , and maybe we could had options for copy code , like a button and maybe even a preview :man_shrugging:

Reply:
I love the copy code idea, but I don't have a good way to do that right now, Code blocking here just seems a little blocky

Reply:
Reference code will be put in the github eventually, I just haven't had the will to write that all out yet.